cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -22,39 +28,48 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.kafka.common.TopicPartitionReplica;
+import org.easymock.EasyMock;
 import org.junit.Test;
 
+import static org.apache.kafka.common.KafkaFuture.completedFuture;
+import static org.easymock.EasyMock.anyObject;
 import static org.junit.Assert.assertEquals;
 import static com.linkedin.kafka.cruisecontrol.common.TestConstants.TOPIC1;
 import static com.linkedin.kafka.cruisecontrol.common.TestConstants.TOPIC2;
+import static org.apache.kafka.clients.admin.DescribeReplicaLogDirsResult.ReplicaLogDirInfo;
 
 /**
  * Unit test class for execution task planner
  */
 public class ExecutionTaskPlannerTest {
+  private ReplicaPlacementInfo _r0 = new ReplicaPlacementInfo(0);
```

Comment:
Can we move the relevant changes to a `master` branch patch?

Reply:
Can we move the broker ids such as `1` to variables as we did here? I.e. let's do the part#2 of: (1) using `ReplicaPlacementInfo`, and (2) moving the hardcoded values to fields in `master`.